besu/src/main/java/org/hyperledger/besu/cli/options/unstable/MergeOptions.java

```diff
@@ -51,7 +49,8 @@ public void consumeParameters(
         final Stack<String> args,
         final CommandLine.Model.ArgSpec argSpec,
         final CommandLine.Model.CommandSpec commandSpec) {
-      RayonismOptions.setMergeEnabled(Boolean.parseBoolean(args.pop()));
+      org.hyperledger.besu.config.experimental.MergeOptions.setMergeEnabled(
```

Comment:
I think we can use an import here

Reply:
Oh man yeah.  That is Intellij refactoring gone wild :)